clienr-rec/src/components/FilterButtons.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { setFilter } from "../features/tasks/taskSlice";

```

Comment:
Please configure and use the absolute paths like `import { setFilter } from "../features/tasks/taskSlice";
import { setFilter } from "@/features/tasks/taskSlice";`